projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.scss

```diff
@@ -0,0 +1,29 @@
+.tabs-viewport {
+    display: flex;
+    height: 20rem;
```

Comment:
Maybe check if something like `1/2vh` or `2/3vh` makes sense.
Would there be a good values that adapts to the screen?